cmake/addons/CMakeLists.txt

```diff
@@ -156,8 +156,11 @@ if(ADDON_SRC_PREFIX)
   message(STATUS "Overriding addon source directory prefix: ${ADDON_SRC_PREFIX}")
 endif()
 
+include(../../cmake/scripts/common/Macros.cmake)
```

Comment:
Maybe use then her also like on line 83, with:
`include(${CORE_SOURCE_DIR}/cmake/scripts/common/Macros.cmake)`

Reply:
Thx, I tried different other variables but somehow missed this one...